docs/topics/mta-view-questionnaire.adoc

```diff
@@ -5,13 +5,13 @@
 
 :_content-type: PROCEDURE
 [id="mta-view-questionnaire_{context}"]
-= Viewing a questionnaire
+= Viewing an assessment questionnaire
 
-.Procedure
+You can display the questions contained in the assessment questionnaire with the answer choices and weight displayed in the drop-down menu.
 
-To view a questionnaire:
+.Procedure
 
 . In the *Administration* view, select *Assessment questionnaires*.
-. Click the Options menu {kebab}.
-. Select *View*.
-. The question contained in the questionnaire are displayed, with the answer choices and weigth displayed in the drop-down option.
+. Click the Options menu ({kebab}).
+. Select *View* for the questionnaire you want to display.
+. Optional: In the drop-down menu, display the answer choices and weight for the chosen questionnaire.
```

Comment:
Replace the above statement with this :
The questions are expandable . Click the left arrow to view  answer choices and their associated risk weight.

Reply:
You mean the first sentence? "The question contained...."?

Reply:
Replace
Optional: In the drop-down menu, display the answer choices and weight for the chosen questionnaire.
with 
Optional : The questions are expandable . Click the left arrow to view answer choices and their risk weight.

Reply:
@nachandr, "left arrow" sounds a bit confusing. Does this arrow have a name, maybe? Or can we say something like "Click the arrow to the left from _<something>_ to display answer choices...." instead? The documentation must be as specific as possible.

Reply:
"Click the arrow to the left from to display answer choices.."

That sounds good.